sound/soc/codecs/rt711.c

```diff
@@ -1250,7 +1242,9 @@ int rt711_io_init(struct device *dev, struct sdw_slave *slave)
 			rt711_jack_detect_handler);
 	INIT_DELAYED_WORK(&rt711->jack_btn_check_work,
 			rt711_btn_check_handler);
+	mutex_init(&rt711->calibrate_mutex);
```

Comment:
what is the purpose of this mutex? is this to have mutual exclusion between calibration work and the set_jack call? can this really happen?

Reply:
Yes, it really happened on my side, so it needs to have mutual exclusion between calibration work and the set_jack.